pkg/sql/catalog/descpb/privilege.go

```diff
@@ -296,6 +293,13 @@ func (p PrivilegeDescriptor) Validate(id ID) error {
 			return fmt.Errorf("user %s must not have %s privileges on system object with ID=%d",
```

Comment:
this error message isn't good anymore, we know what kind of object it is now

Reply:
Done